### PR TITLE
Readme

### LINKED ISSUE #32: Update README.md

### PR DESCRIPTION
**Pull request by CaliHam**
### What is the feature?
This PR will close #32 
Update our readme.md draft for now, we can re-do the gifs once all changes are made and finalized.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,55 @@
...
+
+## Installation
+1. `fork` this repository and/or `clone` it to local
+2. Once you have cloned the repo, change (cd) into the directory and install the project dependencies. Run `npm install` or `npm i` to install project dependencies.
+3. Run `npm test` to see tests
+4. Run `npm run lint` if you would like to see the linter
+5. Run `npm start` and visit `localhost:8080`
+6. Run `npm start` with Whats Cookin API Starter Kit
+
+## Contributers
```
Comment (paulina-isabel) [inline README.md:43]: Should be 'contributors' :)

**Review by paulina-isabel — APPROVED** [review]

**Review by jalbe0076 — CHANGES_REQUESTED** [review]
Looks great! Minor spelling Paulina caught and title to be discussed. 
Gif's to be added and let's add the deployed link: https://jalbe0076.github.io/whats-cooking/

**Review by alivaditis — APPROVED** [review]
looks awesome!

**Comment by CaliHam** [comment]
Great catches! I fixed the title, spelling error, and added the deployed link.

**Pull request merged**